tests/ext/test_callbackcontext.py

```diff
@@ -211,8 +212,9 @@ def test_drop_callback_data_exception(self, bot, app):
         finally:
             app.bot = bot
 
-    async def test_drop_callback_data(self, bot, monkeypatch, chat_id):
-        app = ApplicationBuilder().token(bot.token).arbitrary_callback_data(True).build()
+    async def test_drop_callback_data(self, bot, chat_id):
+        new_bot = make_bot(token=bot.token, arbitrary_callback_data=True)
+        app = ApplicationBuilder().bot(new_bot).build()
```

Comment:
What is this change good for? 🤔

Reply:
they changed something in the api update and this method was getting flood errors quite often. making use of `make_bot` mitigates that because that uses `NonchalantHTTPXRequest` which xfails flood errors